### ISSUE TITLE
"Error_MobileUnsupportedTemplate" output from user function on desktop

### ISSUE THREAD

**Issue by waltcf** [SilentVoid13/Templater#796]
**Plugin information (please complete the following information):**
-   OS: macOS Mojave 10.14.6
-   Templater version: 1.13.0
-   Obsidian version: 0.15.9
-   Templater settings:
	- Template folder location: Assets/Templates/
	- Timeout: 5
	- Shell binary location: /bin/bash
	- Name of user function not working: NWSdayforecastAZA
	- Command of function not working: curl -s https://api.weather.gov/gridpoints/PSR/172,50/forecast | /usr/local/bin/fx-macos .properties.periods[1].detailedForecast

**Describe the bug**
When any of four user functions with the construction mentioned above run (as called by one of two templates), they immediately insert the text "Error_MobileUnsupportedTemplate."

**Expected behavior**
Function runs successfully and then inserts the text delivered by the command.

**Additional context**
These user functions were working (for the most part) right up until a couple days ago when they started immediately giving the "Error_MobileUnsupportedTemplate" output for the templates below. Other than opening the vault on an additional computer, I can think of nothing that changed. I updated from Templater 1.12.0 to 1.13.0 in an attempt to solve the issue, but it had no effect.

**Comment by shabegom** [comment]
Cool. Probably a side effect of removing some logic around running user templates on mobile.

**Comment by shabegom** [comment]
Hm, here is where the error appears to be firing in the code.
https://github.com/SilentVoid13/Templater/blob/fb5619c17d2158201f177f2fa75834bfe414254c/src/core/functions/user_functions/UserSystemFunctions.ts#L54

@waltcf what type of Mac do you have?

**Comment by waltcf** [comment]
@shabegom It's happened on both my iMac (27-inch, Late 2012) and Mac mini (also Late 2012).

**Comment by waltcf** [comment]
Well, now I feel like an idiot.

I went to create a new template to try that code out that you just shared...and happened to notice that the function callouts in those template files had been replaced with the "Error_MobileUnsupportedTemplate" text. So the functions weren't even being run because they'd been overwritten with static text.

So there's no weird bug after all!

I'm perplexed, though, with how those function callouts got replaced by the error text. Would that happen if I try to view a template file on mobile Obsidian?
